### PR TITLE
Add hangup function to Hikconnect API

### PR DESCRIPTION
**Pull request by mayankraichura**
A new asynchronous function 'hangup' was added to the hikconnect/api.py. The purpose of this function is to send a hangup request to a specific device, using the device's unique serial number.

PS: I'm a python newbie. I just used information available at [forum](https://community.home-assistant.io/t/ds-kd8003-ds-kv8113-ds-kv8213-ds-kv6113-ds-kv8413-and-integration-hikvision-hikconnect-video-intercom-doorbell/238535/1134) and implemented it by copying `unlock` function. I've tested it with my device, and it seems to work as expected.

### CODE REVIEW

File: hikconnect/api.py [hunk 1]
```diff
@@ -252,6 +252,22 @@ async def get_call_status(self, device_serial: str):
...
         }
 
+    async def hangup(self, device_serial: str):
+        """
+        Send hangup request.
+
+        The `device_serial` parameter can be obtained from `get_devices()` and/or `get_cameras()`.
+        """
+        async with self.client.put(
+                f"{self.BASE_URL}/v3/devconfig/v1/call/{device_serial}/operation?cmdId=3&handler={self.client.FEATURE_CODE}"
```
Comment (tomasbedrich) [inline hikconnect/api.py]: Is `&handler={self.client.FEATURE_CODE}` really needed? Can we test what happens if it is omitted?

Comment (mayankraichura) [reply hikconnect/api.py]: @tomasbedrich I can confirm that `&handler={self.client.FEATURE_CODE}` is not required and can be safely removed from the URL.

Comment (mayankraichura) [reply hikconnect/api.py]: Also, as per [this post](https://community.home-assistant.io/t/ds-kd8003-ds-kv8113-ds-kv8213-ds-kv6113-ds-kv8413-and-integration-hikvision-hikconnect-video-intercom-doorbell/238535/1149), command `3` is cancel call and so I'll rename the function to `cancelCall`.

Comment (tomasbedrich) [reply hikconnect/api.py]: Agree, just please use `cancel_call` (snake case) as per Python convention.

Comment (mayankraichura) [reply hikconnect/api.py]: @tomasbedrich I've updated the function to cancel_call. Please review let me know if you need anything.

**Pull request merged**